src/main/java/at/ac/tuwien/damap/rest/persons/MockUniversityPersonServiceImpl.java

```diff
@@ -1,25 +1,35 @@
 package at.ac.tuwien.damap.rest.persons;
 
-import at.ac.tuwien.damap.rest.dmp.domain.ContributorDO;
-import io.quarkus.arc.DefaultBean;
-import org.eclipse.microprofile.rest.client.inject.RestClient;
+import java.util.List;
 
 import javax.enterprise.context.ApplicationScoped;
-import javax.inject.Inject;
-import java.util.List;
+
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+
+import at.ac.tuwien.damap.rest.dmp.domain.ContributorDO;
+import io.quarkus.arc.DefaultBean;
+import lombok.extern.jbosslog.JBossLog;
 
 /*
     extend this class in your custom project, for your implementation
  */
 
 @ApplicationScoped
 @DefaultBean
-public class MockPersonServiceImpl implements PersonService {
+public class MockUniversityPersonServiceImpl implements PersonService {
 
-    @Inject
     @RestClient
     MockPersonRestService mockPersonRestService;
 
+    public MockUniversityPersonServiceImpl() {
+        // manual creation of service should take care of creating injected values.
```

Comment:
there should be no need to circumvent Injection, instead we should suppress building this mock service in production environments.